core/modules/widgets/link.js

```diff
@@ -207,7 +207,10 @@ Selectively refreshes the widget if needed. Returns true if the widget or any of
 */
 LinkWidget.prototype.refresh = function(changedTiddlers) {
 	var changedAttributes = this.computeAttributes();
-	if(changedAttributes.to || changedTiddlers[this.to] || changedAttributes["aria-label"] || changedAttributes.tooltip) {
+	var hideMissingLinks = (this.getVariable("tv-show-missing-links") || "yes") === "no";
+	if(changedAttributes.to || changedTiddlers[this.to] || changedAttributes["aria-label"] || changedAttributes.tooltip ||
+		changedAttributes["class"] || changedAttributes.tabindex || changedAttributes.draggable || changedAttributes.tag ||
+		hideMissingLinks !== this.hideMissingLinks) {
```

Comment:
I'm not sure about line 210 and 213. ... If the `tv-show-missing-links` variable changes, the "refresh" from eg: the vars or set-widget will be run. Right? ... So the link-widget should have been refreshed, with a new this.hideMissingLinks variable. So I think 210 and 213 should be not necessary. 

I didn't run tests, so it's only a guess

Reply:
Hi @BurningTreeC that is correct: we don't need to worry about variable updates because a variable update will always refresh the widget tree from that point.

Reply:
Ok thanks @pmario @Jermolene, I've changed it